docs/source/methods/cross_sections.rst

```diff
@@ -63,6 +63,53 @@ Other Methods
 A good survey of other energy grid techniques, including unionized energy grids,
 can be found in a paper by Leppanen_.
 
+---------------------------------
+Windowed Multipole Representation
+---------------------------------
+
+In addition to the usual pointwise representation of cross sections, OpenMC
+offers support for an experimental data format called windowed multipole (WMP).
+This data format requires less memory than pointwise cross sections, and it
+allows on-the-fly Doppler broadening to arbitrary temperature.
+
+The multipole method was introduced by [Hwang]_ and the faster windowed
+multipole method by [Josey]_.  In the multipole format, cross section resonances
+are represented by poles, `p_j`, and residues, `r_j`, in the complex plane.  The
+0 K cross sections in the resolved resonance region can be computed by summing
+up a contribution from each pole:
+
+.. math::
+   \sigma(E, T=0\text{K}) = \frac{1}{E} \sum_j \text{Re} \left[
+   \frac{i r_j}{\sqrt{E} - p_j} \right]
+
+Assuming free-gas thermal motion, cross sections in the multipole form can be
+analytically Doppler broadened to give (after some approximation) the form:
+
+.. math::
+   \sigma(E, T) = \frac{1}{2 E \sqrt{\xi}} \sum_j \text{Re} \left[i r_j
+   \sqrt{\pi} W(z) \right]
```

Comment:
I would personally like to see discussion about the second term that appears as that was one point that I was confused about looking at the source code. But, I understand you've gone above and beyond by even including this extra documentation (thank you!) so if you're reluctant to go into too much detail that's ok. If you'd rather defer, we can just open an issue for it.